### PR TITLE
Cache the system information metrics

### PR DESCRIPTION
**Pull request by longjiquan**
Signed-off-by: dragondriver <jiquan.long@zilliz.com>

issue: #6249

### CODE REVIEW

File: internal/util/metricsinfo/cache.go [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+	manager.systemInfoMetricsMtx.RLock()
+	defer manager.systemInfoMetricsMtx.RUnlock()
+
+	return (!manager.systemInfoMetricsInvalid) &&
+		(manager.systemInfoMetrics != nil) &&
+		(time.Since(manager.systemInfoMetricsLastUpdatedTime) < retention)
+}
+
+func (manager *MetricsCacheManager) GetSystemInfoMetrics() (*milvuspb.GetMetricsResponse, error) {
+	if !manager.IsSystemInfoMetricsValid() {
```
Comment (czs007) [inline internal/util/metricsinfo/cache.go:87]: do not call this function. just acquire the lock once and check whether it's valid in this function.

Comment (DragonDriver) [reply internal/util/metricsinfo/cache.go]: fixed.

**Pull request merged**